aiida/storage/sqlite_temp/backend.py

```diff
@@ -42,7 +47,8 @@ def create_profile(
         name: str = 'temp',
         default_user_email='user@email.com',
         options: dict | None = None,
-        debug: bool = False
+        debug: bool = False,
+        repo_path: str | Path | None = None,
```

Comment:
Why is this optional? Shouldn't this be required?

Reply:
This will default to the `SandboxFolder` default, i.e. a temporary directory, I felt that was a suitable default no?

Reply:
I see know, just wasn't very clear since that happens in `get_repository`. Fine to keep, but maybe a comment in the docstring of the `create_profile` might be useful.